src/state/slices/portfolioSlice/utils.ts

```diff
@@ -111,8 +111,10 @@ export const accountIdToLabel = (accountId: AccountSpecifier): string => {
 }
 
 // note - this is not really a selector, more of a util
-export const accountIdToFeeAssetId = (accountId: AccountSpecifier): AssetId =>
-  chainIdToFeeAssetId(accountIdToChainId(accountId))
+export const accountIdToFeeAssetId = (accountId: AccountSpecifier): AssetId | undefined => {
```

Comment:
this is probably one place we can safely non-null assert here and always return `AssetId`, as the only way we obtain an accountId is if we have a chain adapter that supports it, and every chain adapter by definition has a fee asset id